### PR TITLE
Updated execution graph look-and-feel

### PR DESCRIPTION
**Pull request by kubama**
image

### CODE REVIEW

File: widgets/executions/widget.css [hunk 1]
```diff
@@ -18,29 +18,27 @@
...
 }
 
 .rect-tasks-graph-succeeded {
-    fill: rgba(25, 200, 54, 0.836);
+    fill: rgb(39, 230, 109);
 }
 
 .rect-tasks-graph-failed {
-    fill: rgba(204, 50, 50, 0.822)
+    fill: rgb(204, 50, 50);
```
Comment (qooban) [inline widgets/executions/widget.css]: From where did you take the colors?

File: widgets/executions/src/backend.js [hunk 2]
```diff
@@ -102,7 +102,7 @@ module.exports = r => {
         // This will be used when a text needs to be displayed inside a node (rectangle)
         // and exceeds its width, resulting in increasing the rectangle's height and
         // breaking the string into 2 (and so forth...)
-        const textSizingFactor = 5.6;
+        const textSizingFactor = 5.8;
```
Comment (qooban) [inline widgets/executions/src/backend.js]: That change is to prevent overlapping the rectangle borders by the text, right? Will it cover all cases?

Comment (kubama) [reply widgets/executions/src/backend.js]: It won't cover all cases, it just reduces the possibility.
To cover all cases we would need to set this const to the width of broadest character. But then in an average case we would split the lines very early, in about half of the available line length.

File: widgets/executions/widget.css (hunk 1 above)
Comment (kubama) [reply widgets/executions/widget.css]: From WebStorm color picker :-)

Comment (qooban) [reply widgets/executions/widget.css]: Can we adjust it to be aligned with Semantic-UI colors (main green, yellow, red, ...)?

Comment (kubama) [reply widgets/executions/widget.css]: I find the colors that I used a little bit better suited, for example "official" red and green are to bright and intense and don't look that good when used as a fill for rects that might be much bigger than Semantic components like buttons.

Comment (qooban) [reply widgets/executions/widget.css]: OK, then let's keep it.

File: widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js [hunk 3]
```diff
@@ -114,7 +114,7 @@ export default class ExecutionWorkflowGraph extends React.Component {
     scrollToInProgress() {
         const focusNode = _.find(this.state.graphResult.children, containerNode =>
             _.find(containerNode.children, subGraphNode =>
-                _.includes(['sent', 'rescheduled'], subGraphNode.labels[0].state)
+                _.includes(['Sent', 'Rescheduled'], subGraphNode.labels[0].state)
```
Comment (qooban) [inline widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js]: Let's have an array for all of that task states.
AFAIR we should add 'started' here as well.

Comment (kubama) [reply widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js]: > Let's have an array for all of that task states.

We already have an array.

> AFAIR we should add 'started' here as well.

Added.

File: widgets/executions/src/tasksGraph/GraphNode.js [hunk 4]
```diff
@@ -4,64 +4,87 @@
...
+    let currentTextPlacement_Y = 0;
+
+    const title = labels.display_title || [labels.text];
+    const displayText = labels.display_text;
+
+    const { state } = labels;
+    let stateColor;
+    switch (state) {
+        case 'Rescheduled':
+        case 'Sent':
```
Comment (qooban) [inline widgets/executions/src/tasksGraph/GraphNode.js]: What about having `started` here as well?

File: widgets/executions/src/tasksGraph/GraphNode.js [hunk 5]
```diff
@@ -4,64 +4,87 @@
...
+            stateColor = 'rgb(215,227,45)';
```
Comment (qooban) [inline widgets/executions/src/tasksGraph/GraphNode.js]: I would suggest to assign those strings to some `const`'s and name it after color it represents.

File: widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js [hunk 6]
```diff
@@ -115,7 +116,7 @@ export default class ExecutionWorkflowGraph extends React.Component {
     scrollToInProgress() {
         const focusNode = _.find(this.state.graphResult.children, containerNode =>
             _.find(containerNode.children, subGraphNode =>
-                _.includes(['sent', 'rescheduled'], subGraphNode.labels[0].state)
+                _.includes(['Started', 'Sent', 'Rescheduled'], subGraphNode.labels[0].state)
```
Comment (qooban) [inline widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js]: By array I meant having `const` with those values.

File: widgets/executions/src/tasksGraph/GraphNode.js (hunk 4 above)
Comment (kubama) [reply widgets/executions/src/tasksGraph/GraphNode.js]: Of course - good catch

File: widgets/executions/src/tasksGraph/GraphNode.js (hunk 5 above)
Comment (kubama) [reply widgets/executions/src/tasksGraph/GraphNode.js]: Addressed

File: widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js (hunk 6 above)
Comment (kubama) [reply widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.js]: Addressed

File: widgets/executions/src/tasksGraph/States.js [hunk 7]
```diff
@@ -0,0 +1,5 @@
+export default {
+    inProgress: ['Started', 'Sent', 'Rescheduled'],
+    succeeded: ['Succeeded'],
+    failed: ['Failed']
+};
```
Comment (qooban) [inline widgets/executions/src/tasksGraph/States.js]: That makes it clear now. :+1:

**Comment by kubama** [comment]
> Could you please add some recent screenshots?

Done

**Pull request merged**